Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Explosives/breaching_charge.yml

```
@@ -1,3 +1,6 @@
+- type: Tag
+    id: RMCExplosiveBreachingCharge
```

This won't get read properly if the indentation differs
```suggestion
- type: Tag
  id: RMCExplosiveBreachingCharge
```